src/styles/components/chat/_chat-item-card-information-card.scss

```diff
@@ -13,118 +13,85 @@
                 top: 0;
+                left: 0;
                 width: 100%;
                 height: 100%;
-                box-sizing: border-box;
-                background: var(--mynah-bg-gradient);
-            }
-
-            > .mynah-ui-icon {
-                margin-top: var(--mynah-sizing-1);
-                font-size: var(--mynah-font-size-large);
+                background-color: currentColor;
+                opacity: 10%;
+                pointer-events: none;
             }
 
-            > .mynah-chat-item-information-card-header {
-                flex: 1;
-                display: flex;
-                position: relative;
-                box-sizing: border-box;
-                flex-flow: column nowrap;
-                align-items: flex-start;
-                justify-content: flex-start;
-                gap: var(--mynah-sizing-1);
-
-                > .mynah-chat-item-information-card-title {
-                    font-weight: bold;
-                    flex: 0 1 0%;
-                }
-                > .mynah-chat-item-information-card-description {
-                    flex: 1 0 100%;
-                    font-size: var(--mynah-font-size-xxsmall);
+            @each $status in $mynah-statuses {
+                &.status-#{$status} {
+                    color: var(--mynah-color-status-#{$status});
                 }
             }
         }
-    }
-
-    .mynah-chat-item-information-card-footer {
-        display: flex;
-        position: relative;
-        box-sizing: border-box;
-        width: 100%;
-        max-width: 100%;
-        flex-flow: row nowrap;
-        align-items: flex-start;
-        justify-content: flex-start;
-        overflow: hidden;
-        padding: var(--mynah-sizing-3);
-        transition: var(--mynah-short-transition-rev);
-        gap: var(--mynah-sizing-3);
-        border: 1px solid currentColor;
-        border-radius: 0 0 var(--mynah-input-radius) var(--mynah-input-radius);
-
-        > .mynah-ui-icon {
-            font-size: var(--mynah-font-size-medium);
-            color: currentColor;
-        }
-
-        > p {
-            flex: 1;
-            font-size: var(--mynah-font-size-xsmall);
-            margin: 0;
-            color: var(--mynah-color-text-default);
-        }
-
-        &::before {
-            content: '';
-            position: absolute;
-            top: 0;
-            left: 0;
-            width: 100%;
-            height: 100%;
-            background-color: currentColor;
-            opacity: 10%;
-        }
-
-        @each $status in $mynah-statuses {
-            &.status-#{$status} {
-                color: var(--mynah-color-status-#{$status});
-            }
-        }
-    }
-
-    &:has(.mynah-chat-item-information-card-footer) {
         > .mynah-chat-item-information-card-main {
             border-bottom-left-radius: 0;
             border-bottom-right-radius: 0;
-            border-bottom: 0;
+            border-bottom: none !important;
```

Comment:
Any difference between `0` and `none`?

Reply:
Correct way to do it is none. Depending on GPU, it can still render but without a width. Both will not be visible, however the none will skip the rendering part completely.